apps/omg_watcher/test/test_helper.exs

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ExUnit.configure(exclude: [common: true, integration: true, property: true, wrappers: true])
+ExUnit.configure(exclude: [watcher: true, common: true, integration: true, property: true, wrappers: true])
```

Comment:
watcher doesn't test watcher?

Reply:
these are tags for tests. so for example, when you run tests:
`mix test` you don't run tests that are tagged: `[watcher: true, common: true, integration: true, property: true, wrappers: true]`

If we look at our circleci config, it looks similar to this:
```
test:
	mix test --include test --exclude common --exclude watcher --exclude watcher_info --exclude child_chain

test-watcher:
	mix test --include watcher --exclude watcher_info --exclude child_chain --exclude common --exclude test

test-watcher_info:
	mix test --include watcher_info --exclude watcher --exclude child_chain --exclude common --exclude test

test-common:
	mix test --include common --exclude child_chain --exclude watcher --exclude watcher_info --exclude test

test-child_chain:
	mix test --include child_chain --exclude common --exclude watcher  --exclude watcher_info --exclude test
```

so based on the parallel job, it's excluding some and including one specific tag. PLS validate my statements.